pandas/core/categorical.py

```diff
@@ -460,6 +445,37 @@ def _get_levels(self):
     # TODO: Remove after deprecation period in 2017/ after 0.18
     levels = property(fget=_get_levels, fset=_set_levels)
 
+    _ordered = None
+
+    def _set_ordered(self, value):
+        """ Sets the ordered attribute to the boolean value """
+        warn("Setting 'ordered' directly is deprecated, use 'set_ordered'", FutureWarning)
+        self.set_ordered(value, inplace=True)
+
+    def set_ordered(self, value, inplace=False):
```

Comment:
I still like "as_ordered/unordered" (or as_nominal/ordinal) more, as that implies that a value is returned and not that it is inplace and "set_ordered" is so near "set_order" which would be the "set to this categories".